apps/frontend/src/app/Solver/GOSolver/solveLP_simplex.ts

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
Rename the file to solveLP.ts. We don't need to denote the algorithm in the file name as it could change without changing the APIs.